src/outlier_detection/auto_outliers.py

```diff
@@ -121,25 +116,29 @@ def _normal_round(x: float) -> int:
             return f + 1
 
     # Add group count - how many RU refs there are in a cell, perod
-    filtered_df["group_count"] = filtered_df.groupby(groupby_cols)[value_col].transform(
-        "count"
+    filtered_df["group_count"] = (
+        filtered_df.groupby(groupby_cols)[value_col].transform("count") # noqa
     )
 
     # Rank margins
     filtered_df["high"] = filtered_df["group_count"] * upper_clip
     filtered_df["high_rounded"] = filtered_df.apply(
         lambda row: _normal_round(row["high"]), axis=1
     )
-    filtered_df["upper_band"] = filtered_df["group_count"] - filtered_df["high_rounded"]
+    filtered_df["upper_band"] = (
+        filtered_df["group_count"] - filtered_df["high_rounded"] #
+    )
 
     filtered_df["low"] = filtered_df["group_count"] * lower_clip
     filtered_df["lower_band"] = filtered_df.apply(
         lambda row: _normal_round(row["low"]), axis=1
     )
 
     # Ranks of RU refs in each group, depending on their value
-    filtered_df["group_rank"] = filtered_df.groupby(groupby_cols)[value_col].rank(
-        method="first", ascending=True
+    filtered_df["group_rank"] = (
+        filtered_df.groupby(groupby_cols)[value_col].rank( # noqa
```

Comment:
I know it looks strange, but the pre-commits wanted to do horrible stuff with this, and you have to put the # noqa on each line rather than at the end of the statement, I think